scripts/setup_osx.sh

```diff
@@ -129,6 +129,15 @@ if [[ ! " ${VERIFIED_OSX_VERSIONS[@]} " =~ " ${VER} " ]]; then
   done
 fi
 
+# Verify this script is running from the correct folder (root directory)
+dir_err_txt="Please run this script only from the root of the garage \
+repository, i.e. you should run it using the command \
+\"bash scripts/setup_osx.sh\""
+if [[ ! -f setup.py ]] && ! grep -Fq "name='rlgarage'," setup.py; then
```

Comment:
all conditions in an `if` statement must be enclosed in `[[ ]]`. I know the style guide doesn't show this explicitly, but their motivation behind `[[ ]]` is to force all statements in the conjunction into an unambiguous boolean value.

Reply:
I'm getting
`conditional binary operator expected`
when I add the braces around grep.

I can put curly braces around it though.

Reply:
You probably need to use `$()` to evaluate grep in a subshell.

Reply:
```shell
if [[ ! -f setup.py ]] && [[ ! $(grep -Fq "name='rlgarage'," setup.py) ]]; then
  # stuff
fi
```